docs/testnet-Validator.md

```diff
@@ -0,0 +1,164 @@
+# Setup Masternode
+
+## Requirements
+
+The most common way for a beginner to run a validator is on a cloud server running Linux. You may choose whatever [VPS providers](#list-of-vps-providers) that your prefer, and whatever operating system you are comfortable with.
+
+The transactions weights in Galital Testnet were benchmarked on standard hardware. It is recommended that validators run at least the standard hardware in order to ensure they are able to process all blocks in time. The following are not minimum requirements but if you decide to run with less than this beware that you might have performance issue.
+
+### Minimum Hardware :
+
+- 6GB ram, 60 GB Storage, 2 CPU , <strong>stable server uplink connection with fixed IP</strong>
+
+### Ideal Hardware :
+
+- 60GB ram, 300 GB Storage, 6 CPU, <strong>stable server uplink connection with fixed IP</strong>
+
+
+## Using Ubuntu 20.04 : 
+
+### Update your Ubuntu
+```
+sudo apt-get update
+```
+
+### Validator
+
+Install & Configure Network Time Protocol (NTP) Client
+NTP is a networking protocol designed to synchronize the clocks of computers over a network. NTP allows you to synchronize the clocks of all the systems within the network. Currently it is required that validators' local clocks stay reasonably in sync, so you should be running NTP or a similar service. You can check whether you have the NTP client by running:
+
+If you are using Ubuntu 18.04 / 19.04 / 20.04, NTP Client should be installed by default.
+```
+timedatectl
+```
+If NTP is installed and running, you should see System clock synchronized: yes (or a similar message). If you do not see it, you can install it by executing:
+```
+sudo apt-get install ntp
+```
+ntpd will be started automatically after install. You can query ntpd for status information to verify that everything is working:
+```
+sudo ntpq -p
+```
+
+>WARNING: Skipping this can result in the validator node missing block authorship opportunities. If the clock is out of sync (even by a small amount), the blocks the validator produces may not get accepted by the network. This will result in ImOnline heartbeats making it on chain, but zero allocated blocks making it on chain. 
+>
+
+
+## Installing the Galital Testnet Binary
+
+<br>
+
+Install and configure time
+```
+sudo apt install chrony
+```
+
+Enable it
+```
+sudo systemctl enable chrony
+```
+
+Allow the process in firewall
+
+```
+sudo apt install firewalld
```

Comment:
Firewalld isn't default on ubuntu, it's UFW it need to be adapted:
`ufw allow 22 && ufw allow 30333 && ufw enable`

Also always good to install fail2ban:

`apt install -y fail2ban && systemctl enable fail2ban && service fail2ban start `